lib/fileExportReporter.js

```diff
@@ -68,23 +80,37 @@ const FileExportReporter = function(baseReporterDecorator, config, logger) {
 			return;
 		}
 
-		for (const file of result.exportFiles) {
-			const pathSegments = [outputDir, path.basename(file.name)];
-			if (multiBrowsers) {
-				pathSegments.splice(1, 0, path.basename(browser.name));
-			}
+		if (!Array.isArray(result.exportFiles)) {
+			log.warn("Export files must be given as an array");
+			return;
+		}
+
+		let exportPath = outputDir;
+		if (multiBrowsers) {
+			exportPath = path.join(exportPath, escapeFileName(browser.name));
+		}
+		try {
```

Comment:
try-catch should be around the whole function body (onBrowserComplete). Otherwise errors won't be handled.

Reply:
Done